manim/animation/transform.py

```diff
@@ -61,14 +91,15 @@ def __init__(
         self.path_arc_centers: np.ndarray = path_arc_centers
         self.path_arc: float = path_arc
 
-        if self.path_arc_centers is not None:
+        if path_func is not None:
+            self.path_func: Callable = path_func
```

Comment:
At this point i don't question how python works ... but shouldn't this code result in an error if `path_func` is `None` because it is never set ?

Reply:
Scroll down a bit, `path_func` is actually a property that has this `None`-check built-in. It is confusing.

Reply:
It really is confusing but i i guess then it is fine